### ISSUE TITLE
a11y: Missing label for navigation region (SideNav)

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#14122]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

`a11y issue 30`

Accessible name is not specified for the navigation region resulted in a confusing experience for screen reader users (SideNav). Navigation region containing interactive elements such as Starred, Cluster, Nodes, Events and so on

**To Reproduce**

**Result**
Accessible name is not specified for the navigation region resulted in a confusing experience for screen reader users

**Expected Result**

Screen reader users should be presented with accessible names for navigation regions.

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-71a527a9c07db8bcc2ddfe95ae3ed5bd012f4589-head`
UI version: `master 980d4b0` Firefox version: `138`
Chrome version: `136`

Each sidenav element/region was mentioned by the voice over. Region, web element type, visited or not, and the link text.
✅
